inventory-manager.lic

```diff
@@ -335,8 +335,15 @@ class InventoryManager
   def list_character_inv(name)
     if name.nil?
       DRC.message "There is inventory data for:"
-      @item_data.each { |k, v| DRC.message "    #{k.to_s}" }
-      return
+      @item_data.each do |k, v|
+      inv_count = 0
+      # DRC.message "Checking #{k.to_s}:"
+        v.each { |data|
+        inv_count += 1
+        }
+      DRC.message "#{k.to_s} - #{inv_count}"
+    end
+    return  
```

Comment:
Looks like we've got an indentation issue here (line 339 is part of the .each loop), as well as a commented out line (line 340).